scripts/build_shapes.py

```diff
@@ -268,7 +317,9 @@ def eez(countries, geo_crs, country_shapes, EEZ_gpkg, out_logging=False, distanc
     )
 
     ret_df = ret_df.apply(lambda x: make_valid(x))
-    country_shapes = country_shapes.apply(lambda x: make_valid(x))
+
+    # country_shapes may consist of different geometries which need to be united
+    country_shapes = country_shapes.apply(lambda x: make_valid(x)).unary_union
```

Comment:
By doing this don't we merge all country shapes into one?
Does this work for multi-country regions?

Reply:
Yes! That is applied under a loop, country-by-country. Have tested on a two-countries list (["PK", "KG"]) and it seems to work properly